### PR TITLE
[CS-2311]: Revert to show only local time on payment confirmation

### PR DESCRIPTION
**Pull request by danibonilha**
### Description

- [x] Completes #(CS-2311)

### Screenshots

Screen Shot 2021-10-28 at 16 30 05

### CODE REVIEW

File: cardstack/src/utils/date-utils.ts [hunk 1]
```diff
@@ -116,18 +116,11 @@ export const groupAccumulations = (
...
   delimitation = '\n'
 ) => {
-  let timestamp = new Date(ts * 1000);
-
-  if (isLocalTime) {
-    // get local time based on timezone offset
-    const offsetInMinites = new Date().getTimezoneOffset();
-    timestamp = subMinutes(timestamp, offsetInMinites);
-  }
+  const timestamp = new Date(ts * 1000);
```
Comment (topwebtek7) [inline cardstack/src/utils/date-utils.ts:123]: @danibonilha just curious.. did you confirm if it shows local time? I thought it's utc time?

**Review by topwebtek7 — COMMENTED** [review]
looks good

File: cardstack/src/utils/date-utils.ts (hunk 1 above)
Comment (danibonilha) [reply cardstack/src/utils/date-utils.ts:123]: Yes, it was actually always showing the local time, but we did had an hardcoded string saying it was UTC before, I made a merchant payment at 4.29PM BRT and it's reflected in the screenshoot.

Comment (topwebtek7) [reply cardstack/src/utils/date-utils.ts:123]: Cool. hmm can we check this confirmation screen in other timezone device to confirm if it's actual local time for different users as well?

**Review by eshinkawa — APPROVED** [review]

**Review by topwebtek7 — APPROVED** [review]
🚀

**Pull request merged**